src/NgLamVN/InvCraft/menu/CraftMenu.php

```diff
@@ -86,7 +85,8 @@ public function MenuListener(InvMenuTransaction $transaction) : InvMenuTransacti
 				}
 			}
 		}
-		$this->setResult(ItemFactory::getInstance()->get(0));
+		$air = $this->getLoader()->getProvider()->stringToItem(strtolower("Air"));

```

Comment:
You can replace this with `VanillaItems::AIR()`

Reply:
VanillaItems hmm
But what wrong on this ?